src/Resolver.hack

```diff
@@ -26,10 +26,15 @@ final class Resolver {
      * @see https://spec.graphql.org/draft/#sec-Execution
      */
     public async function resolve(
-        \Graphpinator\Parser\ParsedRequest $request,
+        string $input,
```

Comment:
Not really related, but we discussed not having clients need to understanding parsing the request so I implemented that here.